src/adapters/PeopleAdapter.js

```diff
@@ -6,11 +6,26 @@ import {throwError} from 'rxjs';
  * adhering to the exact parameters and structure of the returned objects
  */
 export default class PeopleAdapter {
+  /**
+   * A Person object with details about the person
+   * @typedef {object} PersonObject
+   * @param {string} id The person identifier
```

Comment:
Shouldn't this be `@property` instead? https://jsdoc.app/tags-property.html

Reply:
Yup. Good 👀